src/Sylius/Behat/Page/Shop/Account/AddressBook/IndexPage.php

```diff
@@ -81,6 +81,31 @@ public function deleteAddress($fullName)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function setAsDefault($fullName)
+    {
+        $addressToSetAsDefault = $this->getAddressOf($fullName);
+        $addressToSetAsDefault->pressButton('Set as default');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function hasNoDefaultAddress()
+    {
+        return !$this->hasElement('default_address');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFullNameOfDefaultAddress()
+    {
+        return $this->getElement('default_address')->find('css', 'address > strong')->getText();
```

Comment:
`$this->getElement('default_address')->find()` can return `null` we should `Assert::notNull` before using `->getText()` method.